crates/libcontainer/src/rootfs/utils.rs

```diff
@@ -112,12 +126,64 @@ pub fn parse_mount(m: &Mount) -> (MsFlags, String) {
                 } else {
                     flags |= flag;
                 }
-            } else {
-                data.push(s.as_str());
-            };
+                continue;
+            }
+
+            if let Some((is_clear, flag)) = match s.as_str() {
+                "rro" => Some((false, linux::MOUNT_ATTR_RDONLY)),
+                "rrw" => Some((true, linux::MOUNT_ATTR_RDONLY)),
+                "rnosuid" => Some((false, linux::MOUNT_ATTR_NOSUID)),
+                "rsuid" => Some((true, linux::MOUNT_ATTR_NOSUID)),
+                "rnodev" => Some((false, linux::MOUNT_ATTR_NODEV)),
+                "rdev" => Some((true, linux::MOUNT_ATTR_NODEV)),
+                "rnoexec" => Some((false, linux::MOUNT_ATTR_NOEXEC)),
+                "rexec" => Some((true, linux::MOUNT_ATTR_NOEXEC)),
+                "rnodiratime" => Some((false, linux::MOUNT_ATTR_NODIRATIME)),
+                "rdiratime" => Some((true, linux::MOUNT_ATTR_NODIRATIME)),
+                "rrelatime" => Some((false, linux::MOUNT_ATTR_RELATIME)),
+                "rnorelatime" => Some((true, linux::MOUNT_ATTR_RELATIME)),
+                "rnoatime" => Some((false, linux::MOUNT_ATTR_NOATIME)),
+                "ratime" => Some((true, linux::MOUNT_ATTR_NOATIME)),
+                "rstrictatime" => Some((false, linux::MOUNT_ATTR_STRICTATIME)),
+                "rnostrictatime" => Some((true, linux::MOUNT_ATTR_STRICTATIME)),
+                "rnosymfollow" => Some((false, linux::MOUNT_ATTR_NOSYMFOLLOW)),
+                "rsymfollow" => Some((true, linux::MOUNT_ATTR_NOSYMFOLLOW)),
+                // No support for MOUNT_ATTR_IDMAP yet (needs UserNS FD)
```

Comment:
Sorry, may I ask you to explain in more detail for me?

Reply:
> No support for MOUNT_ATTR_IDMAP yet (needs UserNS FD)
About this comment?

Reply:
Yes 🙏

Reply:
In crun (https://github.com/containers/crun/pull/874) idmap is implemented as mount option.
But in runc (https://github.com/opencontainers/runc/issues/2821#issuecomment-1042955151), there is an 
opinion that it is not appropriate to implement idmap as mount option. So I left a comment as a note.

Reply:
It's not that important, so I can delete this comment.